src/main/g8/project/Projects.scala

```diff
@@ -25,7 +25,7 @@ object Projects extends Build {
     .settings(Testing.settings:_*)
 
   lazy val examples = project.in(file("example"))
-    .dependsOn(core % "compile->compile;test->test")
+    .dependsOn(core)
```

Comment:
They are two different modes of dependency: `dependsOn` creates a classpath dependency. `test->test` is useful if you have a test helper lib and want to share it between modules. `aggregate` creates a sbt task dependency, so that if you type `compile`, it will be run on the current project, and each of its aggregates. This is usually the behavior a user expects when working with multiple modules.